src/Microsoft.Identity.Client/ClientAssertion.cs

```diff
@@ -62,7 +62,7 @@ public ClientAssertion(string signedAssertion)
         public X509Certificate2 Certificate { get; set; }
 
         /// <summary>
-        /// Gets the Signed Assertion
+        /// Gets the signed assertion
```

Comment:
Looks like you can set it too :) Would be good to update all the comments in this class with more detailed descriptions, the existing ones are a bit pointless since they just repeat the name of the properties.

Reply:
The comments here are really placeholders where JM can add additional as necessary. I am not 100% sure what to put here as I do not know the full scope of scenarios that this is intended to cover. On related note, we should also update the comments on UserAssertion as well since they have the same info